src/commonMain/kotlin/fr/acinq/lightning/io/Peer.kt

```diff
@@ -717,7 +717,7 @@ class Peer(
 
                     is ChannelAction.ProcessCmdRes.AddSettledFail -> {
                         val currentTip = currentTipFlow.filterNotNull().first()
-                        when (val result = outgoingPaymentHandler.processAddSettled(actualChannelId, action, _channels, currentTip.first)) {
+                        when (val result = outgoingPaymentHandler.processAddSettled(actualChannelId, action, _channels, currentTip)) {
```

Comment:
The blockheight is needed for auto-retrying payments when we have multiple `WalletParams.trampolineFees`, but we use a single flat fee for trampoline now. We should probably remove the support for multiple fees.